source/install/prod-ubuntu-16.04.rst

```diff
@@ -0,0 +1,489 @@
+..  _prod-ubuntu-1604:
+
+Production Install on Ubuntu 16.04 LTS
+======================================
+
+Install Mattermost in production mode on one, two or three machines, using the following steps:
+
+- `Install Ubuntu Server (x64) 16.04 LTS`_
+- `Set up Database Server`_
+- `Set up Mattermost Server`_
+- `Set up NGINX Server`_
+- `Post-Installation Mattermost Configuration`_
+
+
+Install Ubuntu Server (x64) 16.04 LTS
+-------------------------------------
+
+1. Set up 3 machines with Ubuntu 16.04 with 2GB of RAM or more. The
+   servers will be used for the Proxy, Mattermost (must be
+   x64), and Database.
+
+   -  **Optional:** You can also use a **1 machine setup** (Proxy, Mattermost and Database on one machine) or a **2 machine setup** (Proxy and Mattermost on one machine, Database on another) depending on your data center standards.
+
+2. Make sure the system is up to date with the most recent security
+   patches.
+
+   ::
+
+       sudo apt update
+       sudo apt upgrade
+
+
+Set up Database Server
+----------------------
+
+1.  For the purposes of this guide we will assume this server has an IP
+    address of ``10.10.10.1``.
+2.  Install PostgreSQL 9.5+::
+
+        sudo apt-get install postgresql postgresql-contrib
+
+3.  PostgreSQL created a user account called ``postgres``. You will need
+    to log into that account with::
+
+        sudo -i -u postgres
+
+4.  You can get a PostgreSQL prompt by typing::
+
+        psql
+
+5.  Create the Mattermost database by typing::
+
+        postgres=# CREATE DATABASE mattermost;
+
+6.  Create the Mattermost user by typing::
+
+        postgres=# CREATE USER mmuser WITH PASSWORD 'mmuser_password';
```

Comment:
We should add a note like the one here https://github.com/mattermost/docs/pull/573/files#diff-d3d33d958ceed9b270f0d2b400e98b77R120